react-components/src/architecture/concrete/exampleDomainObject/ExampleView.ts

```diff
@@ -16,13 +16,13 @@ import {
 import { type DomainObjectIntersection } from '../../base/domainObjectsHelpers/DomainObjectIntersection';
 import { WHITE_COLOR } from '../../base/utilities/colors/colorExtensions';
 
-export class ExampleView extends GroupThreeView {
+export class ExampleView extends GroupThreeView<ExampleDomainObject> {
   // ==================================================
   // INSTANCE PROPERTIES
   // ==================================================
 
   public override get domainObject(): ExampleDomainObject {
-    return super.domainObject as ExampleDomainObject;
+    return super.domainObject;
```

Comment:
Is it not better to call the domainObject as his inheritance instead of calling direct from the super class? Or doesnt work in this case for some reason that I'm not following?

Reply:
All override get domainObject() is not necessary anymore when using template in views!

Reply:
You're right - removed them :+1: